WordPress/Classes/ViewRelated/Gutenberg/Layout Picker/GutenbergLayoutPickerViewController.swift

```diff
@@ -27,7 +27,7 @@ class GutenbergLayoutPickerViewController: UIViewController {
     @IBOutlet weak var titleView: UILabel!
     @IBOutlet weak var largeTitleView: UILabel!
     @IBOutlet weak var promptView: UILabel!
-    @IBOutlet weak var categoryBar: UICollectionView!
+    @IBOutlet weak var filterBar: GutenbergLayoutFilterBar!
```

Comment:
Renamed this just to support a more generic naming structure as this might be used for Gutenboarding later.